bundler/lib/bundler/installer/parallel_installer.rb

```diff
@@ -110,12 +117,17 @@ def check_for_unmet_dependencies
 
       warning = []
       warning << "Your lockfile doesn't include a valid resolution."
-      warning << "You can fix this by regenerating your lockfile or trying to manually editing the bad locked gems to a version that satisfies all dependencies."
```

Comment:
I don't think `trying to manually editing` is quite grammatically correct. `trying to manually edit` might work as well, but I didn't think `trying to` helped much here.

This is not strictly related to the rest of this PR, so I can do it in a separate PR if it is distracting.

Reply:
Yeah, if you don't mind extracting this, we can advertise your two PRs as different enhancements in the changelog!

Reply:
👍🏻 Will do. 

That'll also give me another chance at that windows Ruby 3.2 job 😅